apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -86,6 +136,13 @@ function WpcomBlockEditorNavSidebar() {
 		}
 	};
 
+	let listHeading = get( postType, [ 'labels', 'name' ] );
+	listHeading = applyFilters(
```

Comment:
I think this would be better using two constants rather than redefining `listHeading`